example/src/test/index.html.js

```diff
@@ -1,7 +1,15 @@
 import head from 'partials/head'
 import styles from './test.css'
 import Test from 'partials/Test?universal'
+import config from '@kaliber/config'
+import shared from '/partials/shared'
+import SharedComponent from '/partials/SharedComponent?universal'
```

Comment:
leading slash is not necessary

Reply:
we discussed this and decided that using the slash is better: https://github.com/kaliberjs/build/issues/83